src/packages/apollo/src/plugins/cors.ts

```diff
@@ -1,7 +1,13 @@
 import { ApolloServerPlugin } from '@apollo/server';
 import { logger } from '@exogee/logger';
 
-export const Cors: ApolloServerPlugin = {
+export interface CorsPluginOptions {
+	validateOrigin?(origin: string): boolean;
+}
+
+export const corsPlugin = ({
+	validateOrigin = () => false,
```

Comment:
@skeep-exogee do you think the default should be to allow or deny?

Reply:
yeah as it forces us to supply it 👍

Reply:
great, then validate = false is the default we want